poc/vdaf_poc/vdaf.py

```diff
@@ -19,6 +16,7 @@
 PrepState = TypeVar("PrepState")
 PrepShare = TypeVar("PrepShare")
 PrepMessage = TypeVar("PrepMessage")
+F = TypeVar("F", bound=Field)
```

Comment:
Reviewer note: Moved this from below to here. I like the idea of having all type variables for a file in one place.